### PR TITLE
Added ability to change editor by environment variable

### LINKED ISSUE #1: Configurable editing mode
Hej guys,

did some quick experimenting, looks great! One question though: do you think it's possible to somehow change the editing mode of an interactive numba run? The default mode for the prompt-toolkit Application is emacs (https://github.com/facebookincubator/python-nubia/blob/537e6c44834f7c4ecb7b037d7b28edf0bf49925e/nubia/internal/interactive.py#L102), would be sweet to be able to configure this to, uhmm, say, vi :'D
Also eager to do a PR if we come up with a strategy for implementation.

### PR DESCRIPTION
**Pull request by yoavtzelnick**
Closes #1

### CODE REVIEW

**Comment by ylectric** [comment]
@yoavtzelnick Thanks for working on this!
I've one idea: what do you think about using a dedicated environment variable and fall back to `EDITOR` when it's empty? The rationale why I'm bringing this to the table is that people (e.g. myself) may have something Vim-ish or Emacs-en set as their `EDITOR`, but not exactly Vi or Emacs (consider `nvim`).

**CC** @AhmedSoliman 

P. S. Yep, I realise that `EDITOR` should be something simple and I probably shall use `VISUAL` instead, but...

**Comment by yoavtzelnick** [comment]
I believe that's exactly the reason EDITOR is for, and more importantly I hate editing my bashrc for every program :)
What do you say?

File: nubia/internal/interactive.py [hunk 1]
```diff
@@ -97,9 +97,14 @@ def _build_cli(self):
             accept_action=AcceptAction.RETURN_DOCUMENT,
         )
 
+        editor = getattr(EditingMode,
```
Comment (ylectric) [inline nubia/internal/interactive.py]: Could you please add some comments here WRT `VI` and `EMACS` constants so a stranger could understand a magic behind `getattr` and `upper()`?

Comment (yoavtzelnick) [reply nubia/internal/interactive.py]: Done :)

**Comment by yoavtzelnick** [comment]
How do you see the switch happens?

**Pull request closed without merging**